drivers/xen/console/Config.uk

```diff
@@ -1,8 +1,23 @@
-config LIBXEN_CONSOLE
+menuconfig LIBXEN_CONSOLE
 	bool "Xen console drivers"
 	select LIBUKCONSOLE
 	select LIBNOLIBC
 	depends on PLAT_XEN
 	help
 		Drivers for the normal Xen console and
 		the emergency console
+
+if LIBXEN_CONSOLE
+config LIBXEN_CONSOLE_HV
```

Comment:
Hum... I am thinking that we should change this to have two drivers:
One for the emergency console (just hypercalls) and another one for the standard Xen console. The emergency console (due to its simplicity) can also directly register as early console.
This driver split would make these sub configurations unnecessary.